values.yaml

```diff
@@ -263,6 +263,9 @@ scheduler:
   # Number of schedulers to run
   replicas: 1
 
+# Prevent airflow webserver reboot only for Airflow 2.0.0
+webserverReboot: true
```

Comment:
@andriisoldatenko what do you think about nesting the new flag under the existing `webserver` section?

Thinking maybe we could name it something like `webserver.useDefaultAirflowImage` to tie the underlying intention of the flag back to the actual underlying helper name. Just so its a little clearer as to what's actually changing under the hood. What do you think?

Reply:
@schnie done.